spec/models/emergency_message_spec.rb

```diff
@@ -0,0 +1,36 @@
+# == Schema Information
+#
+# Table name: courts
+#
+#  id                   :integer          not null, primary key
+#  message              :text
+#  show                 :boolean          not null, default false
+#
+
+require "spec_helper"
+
+describe EmergencyMessage do
+  before(:each) do
+    EmergencyMessage.destroy_all()
+    @message = create(:emergency_message, :message => "Test message", :show => false)
+  end
+
+
+  describe "fetching" do
+    it "should return 1 result" do
+      EmergencyMessage.first().should == @message
```

Comment:
I am not sure if this expectation adds any value. Maybe if you could explain a little bit what you are trying to test here, I could try to help if needed :)